src/frontend/context/Valutakurs/ValutakursSkjemaContext.tsx

```diff
@@ -186,6 +222,28 @@ const useValutakursSkjema = ({ barnIValutakurs, valutakurs }: IProps) => {
         });
     };
 
+    const erValutakursdatoerLike = () => {
+        const skjemafeltErTomt = skjema.felter.valutakursdato.verdi === undefined;
+        const nyValutakursdatoErTom = valutakurs.valutakursdato == null;
```

Comment:
Her synes jeg det er lettere å lese om det var !!valutakurs.valutakursdato eller eksplisitt undefined/null, fordi det er lett å overse at det faktisk bare er to likhetstegn og ikke tre, og dermed tenke at denne blir true f.eks ved undefined når den egentlig blir false. Det er jo personlig preferanse, men tenkte bare å lufte det 😄